noxfile.py

```diff
@@ -30,8 +30,8 @@
 DEFAULT_PYTHON_VERSION = "3.8"
 
 # We're using two Python versions to test with sqlalchemy 1.3 and 1.4.
-SYSTEM_TEST_PYTHON_VERSIONS = ["3.8", "3.9"]
-UNIT_TEST_PYTHON_VERSIONS = ["3.6", "3.7", "3.8", "3.9"]
+SYSTEM_TEST_PYTHON_VERSIONS = ["3.8", "3.9", "3.10"]
```

Comment:
We really don't need to run systests on all versions:  better would be to have the lower one and the upper one, e.g.:

```python
SYSTEM_TEST_PYTHON_VERSIONS = ["3.8", "3.10"]
```

Reply:
Changed system tests to only run on 3.8 and 3.10